src/Integration.Vsix/SonarLintTagger/AccumulatingIssueConsumer.cs

```diff
@@ -70,19 +70,13 @@ internal class AccumulatingIssueConsumer : IIssueConsumer
 
         public delegate void OnIssuesChanged(IEnumerable<IssueMarker> issueMarkers);
 
-        public AccumulatingIssueConsumer(ITextSnapshot analysisSnapshot, string analysisFilePath, OnIssuesChanged onIssuesChangedCallback)
-            : this(analysisSnapshot, analysisFilePath, onIssuesChangedCallback, new IssueToIssueMarkerConverter())
-        {
-        }
-
-        internal /* for testing */ AccumulatingIssueConsumer(ITextSnapshot analysisSnapshot, string analysisFilePath, OnIssuesChanged onIssuesChangedCallback,
-            IIssueToIssueMarkerConverter issueMarkerConverter)
+        public AccumulatingIssueConsumer(ITextSnapshot analysisSnapshot, string analysisFilePath, OnIssuesChanged onIssuesChangedCallback, IIssueToIssueMarkerConverter issueMarkerConverter)
```

Comment:
The `IIssueToIssueMarkerConverter` is now more complicated than it was before so unfortunately it can't be created directly anymore.